### PR TITLE
fix(sui): test with signature

### PR DESCRIPTION
**Pull request by ewangbitgo**
Ticket: BG-00000

### CODE REVIEW

File: modules/sdk-coin-sui/src/lib/transactionBuilder.ts [hunk 1]
```diff
@@ -159,6 +159,7 @@ export abstract class TransactionBuilder<
   /** @inheritdoc */
   validateKey(key: BaseKey): void {
     try {
+      console.log(key);
```
Comment (margueriteblair) [inline modules/sdk-coin-sui/src/lib/transactionBuilder.ts:162]: also do we need?

File: modules/sdk-coin-sui/test/unit/transactionBuilder/transferBuilder.ts [hunk 2]
```diff
@@ -85,5 +89,120 @@ describe('Sui Transfer Builder', () => {
       };
       should(() => builder.gasData(invalidGasPayment)).throwError('Invalid payment, invalid or missing version');
     });
+
+    it('should submit a transaction', async () => {
```
Comment (margueriteblair) [inline modules/sdk-coin-sui/test/unit/transactionBuilder/transferBuilder.ts:93]: should we comment this out so we don't submit a transaction every time the unit tests run?

**Review by aarvay — COMMENTED** [review]
flushing from review queue..

**Review by LeoKotschenreuther — COMMENTED** [review]
Automatic comment to flush this PR from my review queue. 🚽🚽🚽
Please re-request another review from the respective team(s) when you are ready for reviews.

**Pull request closed without merging**